It makes sense to write it this way, so I think this needs a NOLINT to silence clang-tidy:
```
/github/workspace/src/particles/particle_types.hpp:375:51: warning: repeated branch body in conditional chain [bugprone-branch-clone]
  375 |         if constexpr (particleType == ParticleType::Rad) {
      |                                                          ^
/github/workspace/src/particles/particle_types.hpp:377:3: note: end of the original
  377 |         }
      |          ^
/github/workspace/src/particles/particle_types.hpp:379:56: note: clone 1 starts here
  379 |         else if constexpr (particleType == ParticleType::CIC) {
      |                                                               ^
/github/workspace/src/particles/particle_types.hpp:381:61: note: clone 2 starts here
  381 |         } else if constexpr (particleType == ParticleType::CICRad) {
      |                                                                    ^
/github/workspace/src/particles/particle_types.hpp:386:59: note: clone 3 starts here
  386 |         } else if constexpr (particleType == ParticleType::Sink) {
      |                                                                  ^
261099 warnings generated.
```

--- a/src/particles/particle_types.hpp
+++ b/src/particles/particle_types.hpp
@@ -373,18 +373,18 @@ template <ParticleType particleType, typename problem_t> auto getParticleIntComp
 	amrex::Vector<std::string> names;
 
 	if constexpr (particleType == ParticleType::Rad) { // NOLINT
-		// No integer components
+							   // No integer components
 	}
 #if AMREX_SPACEDIM == 3
-	else if constexpr (particleType == ParticleType::CIC) { // NOLINT
-		// No integer components
+	else if constexpr (particleType == ParticleType::CIC) {	     // NOLINT
+								     // No integer components
 	} else if constexpr (particleType == ParticleType::CICRad) { // NOLINT
-		// No integer components
+								     // No integer components
 	} else if constexpr (particleType == ParticleType::StochasticStellarPop) {
 		const std::vector<std::string> enum_names = amrex::getEnumNameStrings<StochasticStellarPopParticleIntIdx>();
 		names = {enum_names.begin(), enum_names.end()};
 	} else if constexpr (particleType == ParticleType::Sink) { // NOLINT
-		// No integer components
+								   // No integer components
 	} else if constexpr (particleType == ParticleType::Test) {
 		const std::vector<std::string> enum_names = amrex::getEnumNameStrings<TestParticleIntIdx>();
 		names = {enum_names.begin(), enum_names.end()};

--- a/src/particles/particle_types.hpp
+++ b/src/particles/particle_types.hpp
@@ -372,18 +372,18 @@ template <ParticleType particleType, typename problem_t> auto getParticleIntComp
 {
 	amrex::Vector<std::string> names;
 
-	if constexpr (particleType == ParticleType::Rad) {
+	if constexpr (particleType == ParticleType::Rad) { // NOLINT
 		// No integer components
 	}
 #if AMREX_SPACEDIM == 3
-	else if constexpr (particleType == ParticleType::CIC) {
+	else if constexpr (particleType == ParticleType::CIC) { // NOLINT
 		// No integer components
-	} else if constexpr (particleType == ParticleType::CICRad) {
+	} else if constexpr (particleType == ParticleType::CICRad) { // NOLINT
 		// No integer components
 	} else if constexpr (particleType == ParticleType::StochasticStellarPop) {
 		const std::vector<std::string> enum_names = amrex::getEnumNameStrings<StochasticStellarPopParticleIntIdx>();
 		names = {enum_names.begin(), enum_names.end()};
-	} else if constexpr (particleType == ParticleType::Sink) {
+	} else if constexpr (particleType == ParticleType::Sink) { // NOLINT
 		// No integer components
 	} else if constexpr (particleType == ParticleType::Test) {
 		const std::vector<std::string> enum_names = amrex::getEnumNameStrings<TestParticleIntIdx>();

